<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockForm</class>
 <widget class="QWidget" name="StockForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lnum_LWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>61</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="MSG_TView">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>611</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="kind_CBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分类</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>ADMUI3Lg</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 12, 16);</string>
   </property>
   <property name="text">
    <string>数量不足报警</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_Btn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="purchase_Btn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>进货</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="inum_SBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="msg_LEdit">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>相关信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
